README.md

```diff
@@ -42,6 +42,10 @@ import ical from 'ical-generator';
 import http from 'node:http';
 
 const calendar = ical({name: 'my first iCal'});
+
+// A method is required for outlook to display the invitation properly
```

Comment:
```suggestion
// A method is required for outlook to display event as an invitation
```

Since you don't always want to display an invitation, I would rather word it that way. In most cases that I have seen, only entries in the calendar should be displayed. Without rendering an invitation for it.

Reply:
Yes, sure! You would have a better idea of how others use your package. I just want to save people some bug hunting when trying to display the event as an invitation.